gpu_serving/predict.py

```diff
@@ -0,0 +1,46 @@
+import argparse
```

Comment:
This is a CLI for the example?

Reply:
Yes, see README: `python predict.py --url=YOUR_KF_HOST/models/coco`